### ISSUE TITLE
Consider replacing Typescript in favour of ES6

### ISSUE THREAD

**Issue by pigeonfresh** [mediamonks/muban#60]
I know a lot of people are fans and biased about the use of Typescript and I know its benefits. But with more and more people joining, without a background in Typescript I get often the question why Muban comes with Typescript. Would it be a bad thing to see if we could switch to a version without TS. As far as I know we're not using it in any of our other frameworks. I know that Vue is moving towards a version with Typescript.

**Comment by ThaNarie** [comment]
You are free to use es6 in all your files within the project, just update the templates :)
I guess this is mostly about the defaults in the skeleton?

**Comment by pigeonfresh** [comment]
Yes, I guess it hints people in the direction of using Typescript (it least it did for me). So Muban has technically the option to use ES6 instead of TS. Then we should have the discussion on company level about what we would prefer.

**Comment by nossienl** [comment]
Why discussion on company level ? It's just a personal preference. if you want to use es6 go ahead if it doesn't matter u have TS as default.
